langwatch/src/server/tracer/types.ts

```diff
@@ -222,7 +222,7 @@ export type Trace = {
     topic_id?: string;
     subtopic_id?: string;
   };
-  timestamps: { started_at: number; inserted_at: number };
+  timestamps: { started_at: number; inserted_at: number; updated_at: number };
```

Comment:
oh, nice one

Reply:
let's just pay attention when deploying this live, specially if it doesn't break receiving new traces, I don't think it will break anything, but since this is used everywhere better keep a close eye immediately after deploy